src/modules/items/cqrs/queries/handlers/get-item-by-id.query.handler.ts

```diff
@@ -0,0 +1,34 @@
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { GetItemByIdQuery } from '../implements/get-item-by-id.query';
+import { BadRequestException, Inject, NotFoundException } from '@nestjs/common';
+import { ItemRepository } from 'src/modules/items/repository/item.repository';
+import { GetItemByIdResponseDto } from 'src/modules/items/dto/get-item-by-id.response.dto';
+import { isUuid } from 'uuidv4';
+
+@QueryHandler(GetItemByIdQuery)
+export class GetItemByIdQueryHandler
+  implements IQueryHandler<GetItemByIdQuery>
+{
+  constructor(
+    @Inject(ItemRepository)
+    private readonly itemRepository: ItemRepository,
+  ) {}
+
+  async execute(query: GetItemByIdQuery): Promise<GetItemByIdResponseDto> {
+    if (isUuid(query.id) === false) {
```

Comment:
```suggestion
    if (!isUuid(query.id)) {
```

Nhưng đáng ra thì validate không phải ở đây